Fix ens names in chat

cammellos opened the pull request:
We now store the clock value of the message and check against it, to make sure we don't verify old usernames.
status: ready

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | d9fafcfa | [#2](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-9922/2/) | 2020-01-27 12:48:32 | ~10 min | `ios` | [:package:`ipa`](https://i.diawi.com/faZpNj) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2FfaZpNj)|

cammellos committed:
Verify ENS name only if newer messages are sent

We use clock-value to verify ENS messages, so that changing the ENS name
and fetching from mailserver does not result in the name being resolved
again.

yenda commented on src/status_im/ens/core.cljs:
```diff
@@ -278,19 +278,25 @@
+
+(fx/defn verify-names-from-message [cofx {:keys [content clock-value]} signature]
+  (when (should-be-verified? cofx (:ens-name content) signature clock-value)
     {::verify-names [{:name (:ens-name content)
```
it only ever checks one name at a time though?

cammellos replied:
it's accumulated, through `fx/merge`, and then we aggregate them and send a single request.
Currently though, we process one message at a time, so it would only send a request at a time

flexsurfer commented:
@cammellos is this still relevant?

cammellos commented:
@flexsurfer no, forgot about this one, thanks!
